.schema/config.schema.json

```diff
@@ -819,6 +850,15 @@
               ]
             }
           }
+        },
+        "grant": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "jwt": {
+              "$ref": "#/definitions/grantJWT"
```

Comment:
Unless this is being used somewhere else there isn't really a need to have it as a definition so you can just embed it here IMO.